adapter/internal/oasparser/model/resource.go

```diff
@@ -34,16 +34,12 @@ import (
 // mentioned under pathItem.
 type Resource struct {
 	path                string
-	methods             []Operation
+	methods             []*Operation
```

Comment:
Why are we switching to a pointer?

Reply:
because we later sanitizes populated operations and resources in `SanitizeAPISecurity` line 327.